webdriver/tests/bidi/input/perform_actions/key.py

```diff
@@ -41,10 +41,13 @@ async def test_key_down_closes_browsing_context(
         .key_up("w")
     )
 
+    actions_command_future = bidi_session.input.perform_actions(
```

Comment:
I think I would prefer opening the context via window.open. This pattern is potentially flaky as we do not know if by the time bidi_session.browsing_context.close reaches the server the action has already completed or not (e.g., very slow test runner and very fast driver/browser implementation)

Reply:
changed to `window.open`